configs/katello/3.16.yaml

```diff
@@ -99,45 +98,6 @@
       - test-flat-el8
       - centos8-devel
       - puppetlabs-puppet6-rhel-8
-    build_groups:
```

Comment:
we set build_package_group_source_tag, no need to add build_groups too

Reply:
Is this something we can do in nightly and foreman as well?

Reply:
foreman yes, nightly no, as that's the source of truth.